README.md

```diff
@@ -1,212 +1,217 @@
-# RHTAS Ansible Collection
+# Red Hat Trusted Artifact Signer Ansible collection
 
-Automation to deploy the RHTAS ecosystem on RHEL
+The purpose of this Ansible collection is to automate the deployment of the Red Hat Trusted Artifact Signer (RHTAS) service on Red Hat Enterprise Linux (RHEL).
 
-:warning: **The contents of this repository are a Work in Progress.**
+> [!WARNING]
+Currently, the contents of this repository are a work in progress and can change often.
```

Comment:
I think we should just go ahead and remove this warning, because we'll release soon and we don't want this to be displayed on Automation Hub. Maybe we should replace it with a note that this is a tech preview?

Reply:
Done.